added the required attributes - rhidp-3968 (#491)
redhat-developer/red-hat-developers-documentation-rhdh: artifacts/attributes.adoc

:LinkPluginsGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/configuring_plugins_in_red_hat_developer_hub/index
:NameOfPluginsGuide: Configuring plugins in {product}

> :authentication-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index

Reviewer: ```suggestion
:install-on-ocp-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index
```

> :authentication-book-title: Installing Red Hat Developer Hub on OpenShift Container Platform

Reviewer: ```suggestion
:install-on-ocp-title: Installing Red Hat Developer Hub on OpenShift Container Platform
```

> :install-on-ocp-book-title: Installing Red Hat Developer Hub on OpenShift Container Platform

Reviewer: ```suggestion
:install-on-ocp-book-title: Installing {product} on {ocp-short}
```

> :authorization-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index

Reviewer: ```suggestion
:install-on-eks-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index
```

> :authorization-book-title: Installing Red Hat Developer Hub on Amazon Elastic Kubernetes Service

Reviewer: ```suggestion
:install-on-eks-title: Installing Red Hat Developer Hub on Amazon Elastic Kubernetes Service
```

> :install-on-eks-book-title: Installing Red Hat Developer Hub on Amazon Elastic Kubernetes Service

Reviewer: ```suggestion
:install-on-eks-book-title: Installing {product} on {eks-brand-name}
```

> :authorization-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index

Reviewer: ```suggestion
:install-on-aks-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
```

> :authorization-book-title: Installing Red Hat Developer Hub on Microsoft Azure Kubernetes Service

Reviewer: ```suggestion
:install-on-aks-title: Installing Red Hat Developer Hub on Microsoft Azure Kubernetes Service
```

> :install-on-aks-book-title: Installing Red Hat Developer Hub on Microsoft Azure Kubernetes Service

Reviewer: ```suggestion
:install-on-aks-book-title: Installing {product} on {aks-brand-name}
```

> :authorization-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index

Reviewer: ```suggestion
:install-on-airgap-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
```

> :authorization-book-title: Installing Red Hat Developer Hub in an air-gapped environment

Reviewer: ```suggestion
:install-on-airgap-title: Installing Red Hat Developer Hub in an air-gapped environment
```

> :install-in-air-gap-book-title: Installing Red Hat Developer Hub in an air-gapped environment

Reviewer: ```suggestion
:install-in-air-gap-book-title: Installing {product} in an air-gapped environment
```

> 
> :install-on-ocp-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index
> :install-on-ocp-book-title: Installing {product} on {ocp-short}
> 
> :install-on-eks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index
> :install-on-eks-book-title: Installing {product} on {eks-brand-name}
> 
> :install-on-aks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
> :install-on-aks-book-title: Installing {product} on {aks-brand-name}
> 
> :install-in-air-gap-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
> :install-in-air-gap-book-title: Installing {product} in an air-gapped environment
> 
> :upgrade-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
> :upgrade-book-title: Upgrading {product}

Reviewer 2: Consider keeping the gerund and adding the Authentication and Authorization books:

```suggestion

:installing-on-ocp-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index
:installing-on-ocp-book-title: Installing {product} on {ocp-short}

:installing-on-eks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index
:installing-on-eks-book-title: Installing {product} on {eks-brand-name}

:installing-on-aks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
:installing-on-aks-book-title: Installing {product} on {aks-brand-name}

:installing-in-air-gap-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
:installing-in-air-gap-book-title: Installing {product} in an air-gapped environment

:authentication-book-url: https://docs.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/authentication/index
:authentication-book-title: Authentication

:authorization-book-url: https://docs.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/authorization/index
:authorization-book-title: Authorization

:upgrading-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
:upgrading-book-title: Upgrading {product}
```

> :authorization-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index

Reviewer: ```suggestion
:upgrade-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
```

> :authorization-book-title: Upgrading Red Hat Developer Hub

Reviewer: ```suggestion
:upgrade-book-title: Upgrading Red Hat Developer Hub
```

> :upgrade-book-title: Upgrading Red Hat Developer Hub

Reviewer: ```suggestion
:upgrade-book-title: Upgrading {product}
```

:LinkPluginsGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/configuring_plugins_in_red_hat_developer_hub/index
:NameOfPluginsGuide: Configuring plugins in {product}

> :authentication-book-url: https://docs.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/authentication/index

Reviewer 3: Where does `/us-en/` come from?

https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/authentication/index works as does https://docs.redhat.com/us-en/documentation/red_hat_developer_hub/1.2/html/authentication/index and https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html-single/authentication/index

https://docs.redhat.com/us-en/documentation/red_hat_developer_hub/1.2/html-single/authentication/index returns a 404.

I wonder if removing `/us-en/` would resolve the issue. `/us-en/` is not present in any of the other URLs in this PR.

Reviewer: @themr0c ^
